src/resources/views/columns/image.blade.php

```diff
@@ -0,0 +1,18 @@
+<td>
+  @if( !empty($entry->{$column['src']}) )
+    <a
+      href="{{ $entry->{$column['src']} }}"
+      target="_blank"
+    >
+      <img
+        src="{{ $entry->{$column['src']} }}"
+        style="
+          height: {{ isset($column['height']) ? $column['height'] : "25px" }};
```

Comment:
Why not 26px (or 32px or 256px)? Or why not try to guess the height of the image?

Reply:
I found 25px to be a good sensible default, as it can show a _thumbnail_ without problems in the average height of a row 

![image](https://user-images.githubusercontent.com/1755628/30037759-a99147c0-9183-11e7-90ac-7711465fc6d8.png)

Width auto as default helps keeping the aspect ratio of the _thumbnail_ by default. 

Both properties can be overriden, so the `25px ` and `auto` values are only intended to be sensible defaults.

Reply:
@lloy0076 What would it bring to know the height of the image? I don't see the point.

Reply:
Ok.

Reply:
25px height sounds reasonable to me. This will be a matter of opinion no matter what, but if people want to change it, they will. I like it.